docs/cookbook/recipe_icheck.rst

```diff
@@ -43,3 +43,30 @@ set data attribute ``data-sonata-icheck = "false"`` to this form element::
 .. note::
 
     You have to use false as string! ``"false"``!
+
+When using ``Sonata\AdminBundle\Form\Type\ChoiceFieldMaskType`` (or other types that inherit from it, such as )
```

Comment:
I think we don't need this extra-doc, no ?

Reply:
I have updated it now. I think that the doc is helpful, it took me some time to realize that the attribute needs to be added to the choice elements and not to the outer type

Reply:
You still need to pass icheck => false on choice elements even if we don't check for this in the html then ?

Reply:
yea, the thing is that the icheck js tries to apply itself on all the radio inputs, except the one that have `data-sonata-icheck` set to false, but then using the expanded choice, the attributes to the choice need to be passed independently